src/ds++/Assert.hh

```diff
@@ -286,7 +286,7 @@ std::string verbose_error(std::string const &message);
  *
  * Special code for CUDA.
  *
- * If HAVE_CUDA=ON and USE_CUDA=ON, then alter the behavior of the DbC macros
+ * If __NVCC__  (processing with nvcc) and USE_CUDA=ON, then alter the behavior of the DbC macros
```

Comment:
Please wrap at 80 columns (for the time being).

Reply:
Done, thanks!